src/framework/mod.rs

```diff
@@ -56,6 +62,21 @@ impl HttpApiClient {
             http_client: reqwest::Client::new(),
         }
     }
+
+    pub fn new_custom(
```

Comment:
Let's just call this `new()`. People can call `HttpApiClientConfig::default()` if they just want to pass the default values.

Reply:
Breaking change though :/

Reply:
#versioning

Reply:
but I guess we have so few external users right now that it's OK